okapi-common/src/main/java/org/folio/okapi/common/ModuleId.java

```diff
@@ -79,6 +116,9 @@ public boolean equals(Object that) {
     return compareTo((ModuleId) that) == 0;
   }
 
+  /**
+   * @return hash-code
+   */
```

Comment:
Please do not overwrite the inherited Javadoc (`@Override`) unless the new Javadoc is better.

Reply:
oh nice